src/adittyThakare/stringoperations/Assignment25_Program1.java

```diff
@@ -0,0 +1,38 @@
+/*Assignment - 25 : 20th Sep'2022
+
+Program 1: Print all the repeated word in given string.
+String str = "Maulik Puja Hindavi Shubham Hindavi Shubham Titiksha Swati";
+output : Hindavi 
+         Shubham */
+package adittyThakare.stringoperations;
+
+public class Assignment25_Program1 {
+
+	void printRepeatedWordsInString(String input) {
```

Comment:
instead of two loops, you can have one

String str = "Maulik Puja Hindavi Shubham Hindavi Shubham Titiksha Swati";
    String arr[] = str.split(" ");
    for (int i = 0; i < arr.length; i++) {
      if (str.indexOf(arr[i]) == str.lastIndexOf(arr[i])) {
        System.out.println(arr[i]);
      }
    }